api/test/integration/test_experimental_endpoints.tavern.yaml

```diff
@@ -1,9 +1,87 @@
 ---
-test_name: DELETE /experimental/syscheck
+test_name: DELETE /experimental/rootcheck
 
 marks:
   - base_tests
 
+stages:
+
+  # DELETE /experimental/rootcheck
+  - name: Try to delete rootcheck scans in agent 001
+    request:
+      verify: False
+      method: DELETE
+      url: "{protocol:s}://{host:s}:{port:d}/experimental/rootcheck"
+      headers:
+        Authorization: "Bearer {test_login_token}"
+      params:
+        agents_list: "001"
+    response:
+      status_code: 200
+      json:
+        error: 0
+        data:
+          affected_items:
+            - "001"
+          failed_items: []
+          total_affected_items: 1
+          total_failed_items: 0
+
+  # DELETE /experimental/rootcheck
+  - name: Try to delete rootcheck scans in all agents
```

Comment:
The same as above, check this in any other test that is modified in this PR.

Reply:
Done